### ISSUE TITLE
Package uses undefined getters that have been removed in latest Flutter version 3.22

### ISSUE THREAD

**Issue by Joshua27** [rwbr/flutter_neat_and_clean_calendar#75]
Hi,
I can't compile my app using the calendar package after upgrading to Flutter 3.22 due to the use of undefined getters.
For instance: 
```
/home/****/.pub-cache/hosted/pub.dev/flutter_neat_and_clean_calendar-0.3.16+36/lib/flutter_neat_and_clean_calendar.dart:556:63: Error: The getter 'bodyText2' isn't defined for the class 'TextTheme'.
 - 'TextTheme' is from 'package:flutter/src/material/text_theme.dart' ('/home/*****/*****/flutter/packages/flutter/lib/src/material/text_theme.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'bodyText2'.
    final TextStyle? body1Style = Theme.of(context).textTheme.bodyText2;
```
See also the static analysis issues here:
https://pub.dev/packages/flutter_neat_and_clean_calendar/score

These getters were deprecated an have been removed in Flutter 3.22.

Kind regards,
Joshua

**Comment by rwbr** [comment]
I'll take care of it ASAP

**Comment by Tameflame** [comment]
Yup we're having this issue too. Had to downgrade flutter to fix. Really looking forward to a fix! Loving this package!

**Comment by Joshua27** [comment]
Has been fixed in #76 and release v0.3.19+41
